### PR TITLE
Update External Examples to SciPy2023 version

### LINKED ISSUE #123: Good examples as a pyvista in action example
**Describe what feature you would like added.**

- magnetic field example is added in https://github.com/pyvista/pyvista/pull/3058
- 3D Celestial Bodies (planet3D) is added in https://github.com/pyvista/pyvista/pull/2994 
- CG examples.
-  pyFBS https://pyfbs.readthedocs.io/en/latest/index.html
- GeoVista https://github.com/bjlittle/geovista

### PR DESCRIPTION
**Pull request by tkoyama010**
### Overview

Update External Examples to SciPy2023 version.

resolves #123 

### Details

- See https://github.com/bjlittle/geovista/discussions/343 .

### CODE REVIEW

File: requirements_docs.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 cartopy==0.21.1
 cmocean==2.0
 colorcet==3.0.1
-geovista==0.2.0
+git+https://github.com/bjlittle/geovista@main
```
Comment (bjlittle) [inline requirements_docs.txt:4]: @tkoyama010 Note that I'm releasing `geovista` 0.3.0 this week and it'll contain the extra functionality that you need for the tutorial.

Did you want me to raise a separate PR or issue to update the dependencies when 0.3.0 is available?

Comment (tkoyama010) [reply requirements_docs.txt:4]: Yes. Please. Thanks for your help.

Comment (bjlittle) [reply requirements_docs.txt:4]: No problem. Will do 👍

File: tutorial/05_action/a_lesson_geovista.py [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+# At the Met Office, they are moving to an unstructured cube-sphere mesh
+# which is a cube projected out onto a sphere i.e., there are six panels on the
+# sphere. Each cube-sphere is defined by the number of "cells squared" within
+# each panel e.g., the following example is a C48 cube-sphere, so there are 6 *
+# 48 * 48 cells.
+#
+# To that end, GeoVista has prepared samples for it.
+
+import geovista.samples
+import geovista.theme
```
Comment (tkoyama010) [inline tutorial/05_action/a_lesson_geovista.py:30]: It is worth mentioning that we can define a theme in an external package to give it your favorite look.

File: tutorial/05_action/a_lesson_geovista.py [hunk 3]
```diff
@@ -0,0 +1,358 @@
...
+~~~~~~~~~~~~~~
+
+This is provided by `@bjlittle <https://github.com/bjlittle>`_ in
+`this discussion <https://github.com/bjlittle/geovista/discussions/343>`_
+and modifined by `@tkoyama010 <https://github.com/tkoyama010>`_ .
+"""
+
+###############################################################################
+# You may think that PyVista is a little too abstract for what you want to do.
+# Therefore, we will introduce GeoVista, which was developed as a gateway to
```
Comment (tkoyama010) [inline tutorial/05_action/a_lesson_geovista.py:12]:
```suggestion
# Therefore, we will introduce GeoVista.
# GeoVista intends to be a cartographic gateway into the powerful world of PyVista, and all that it offers.
```

File: requirements_docs.txt (hunk 1 above)
Comment (tkoyama010) [reply requirements_docs.txt:4]: 🚀
```suggestion
geovista==0.3.0
```

Comment (bjlittle) [reply requirements_docs.txt]: @tkoyama010 You can now update this to be 0.3.0 🥳

File: tutorial/05_action/a_lesson_geovista.py [hunk 4]
```diff
@@ -0,0 +1,362 @@
+"""
+Using GeoVista
+~~~~~~~~~~~~~~
+
+This is provided by `@bjlittle <https://github.com/bjlittle>`_ in
+`this discussion <https://github.com/bjlittle/geovista/discussions/343>`_
+and modifined by `@tkoyama010 <https://github.com/tkoyama010>`_ .
```
Comment (akaszynski) [inline tutorial/05_action/a_lesson_geovista.py:7]:
```suggestion
and modified by `@tkoyama010 <https://github.com/tkoyama010>`_ .
```

File: tutorial/05_action/a_lesson_geovista.py [hunk 5]
```diff
@@ -0,0 +1,362 @@
...
+c48 = gv.samples.lfric(resolution="c48")
+
+###############################################################################
+# .. note:: **LFRic - a modelling system fit for future computers**
+#
+#      If you are interested in LFRic, please refer to
+#      `LFRic - a modelling system fit for future computers <https://www.metoffice.gov.uk/research/approach/modelling-systems/lfric>`_ .
+
+###############################################################################
+# Since the `c48` is defined as PolyDdata in PyVista, it can be drawn using
```
Comment (akaszynski) [inline tutorial/05_action/a_lesson_geovista.py:74]:
```suggestion
# Since the `c48` is defined as PolyData in PyVista, it can be drawn using
```

**Review by akaszynski — APPROVED** [review]
This works as-is. It's quite long, perhaps too long for a tutorial, but it's a great demonstration of how to use PyVista for what it's best at: within another library for data science.

Let's merge this ASAP. We're down to the wire and there's a bit more work to be done.

**Pull request merged**